docs/topics/asyncio.rst

```diff
@@ -0,0 +1,51 @@
+===============
+asyncio support
+===============
+
+.. versionadded:: 2.0
+
+Scrapy has partial support for :mod:`asyncio` and
+:ref:`coroutine syntax <async>`.
+
+.. warning:: :mod:`asyncio` support in Scrapy is experimental. Future Scrapy
+             versions may introduce related API and behavior changes without a
+             deprecation period or warning.
+
+Installing the asyncio reactor
+==============================
+
+To enable :mod:`asyncio` support, set the :setting:`TWISTED_REACTOR` setting to
+``'twisted.internet.asyncioreactor.AsyncioSelectorReactor'``.
+
+If you are using :class:`~scrapy.crawler.CrawlerRunner`, you also need to
+install the :class:`~twisted.internet.asyncioreactor.AsyncioSelectorReactor`
+reactor manually. You can do that using
+:func:`~scrapy.utils.reactor.install_reactor`::
+
+    install_reactor('twisted.internet.asyncioreactor.AsyncioSelectorReactor')
+
+
+Coroutine support
+=================
+
+The following callables may be defined as coroutines, and hence use
+:ref:`coroutine syntax <async>` and :mod:`asyncio`, as well as
```

Comment:
`async def` syntax should be unrelated to asyncio; I was hoping you can use it with Twisted as well, e.g. `resp = await treq.get(b"http://example.com")`.

Reply:
TIL :open_mouth:

Reply:
I’ve split out a topic about coroutines.